components/azure-iot/component.mk

```diff
@@ -0,0 +1,113 @@
+#
+# Component Makefile
+#
+ 
+# Component configuration in preprocessor defines
+CFLAGS += -DUSE_LWIP_SOCKET_FOR_AZURE_IOT
+ 
+ 
+COMPONENT_ADD_INCLUDEDIRS :=  \
+pal  \
+pal/inc  \
+sdk/c-utility/inc  \
+sdk/c-utility/inc/azure_c_shared_utility \
+sdk/c-utility/pal/inc \
+sdk/c-utility/pal/freertos \
+sdk/c-utility/pal/generic \
+sdk/iothub_client/inc \
+sdk/serializer/inc \
+sdk/umqtt/inc  \
+sdk/umqtt/inc/azure_umqtt_c                 \
+sdk/deps/parson
+ 
+COMPONENT_OBJS =  \
+sdk/c-utility/pal/freertos/lock.o                \
+sdk/c-utility/pal/dns_async.o     \
+sdk/c-utility/pal/socket_async.o               \
+sdk/c-utility/pal/freertos/threadapi.o    \
+sdk/c-utility/pal/freertos/tickcounter.o                \
+sdk/c-utility/pal/lwip/sntp_lwip.o            \
+sdk/c-utility/pal/tlsio_options.o            \
+sdk/c-utility/adapters/agenttime.o         \
```

Comment:
Minor style issue, could you please indent the lines which follow line continuation? I.e.:
```make
COMPONENT_OBJS =  \
        sdk/c-utility/pal/freertos/lock.o \
        sdk/c-utility/pal/dns_async.o \
        sdk/c-utility/pal/socket_async.o \
```

If you could also remove the spaces between the file name and backslashes to make them look a bit more tidy, that would be perfect!